object/user.go

```diff
@@ -47,11 +47,12 @@ func InitUserManager() {
 }
 
 type User struct {
-	Owner       string `xorm:"varchar(100) notnull pk" json:"owner"`
-	Name        string `xorm:"varchar(100) notnull pk" json:"name"`
-	CreatedTime string `xorm:"varchar(100) index" json:"createdTime"`
-	UpdatedTime string `xorm:"varchar(100)" json:"updatedTime"`
-	DeletedTime string `xorm:"varchar(100)" json:"deletedTime"`
+	Owner       string   `xorm:"varchar(100) notnull pk" json:"owner"`
+	Name        string   `xorm:"varchar(100) notnull pk" json:"name"`
+	CreatedTime string   `xorm:"varchar(100) index" json:"createdTime"`
+	UpdatedTime string   `xorm:"varchar(100)" json:"updatedTime"`
+	DeletedTime string   `xorm:"varchar(100)" json:"deletedTime"`
+	Member      []string `json:"member"`
```

Comment:
You should not add new field. What's member? Is it the groups the user belongs to? We already have groups field